configs/subscriptions.json

```
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Subscription Watch administrator",
+      "description": "Perform any available operation against any Subscription Watch resource.",
+      "system": true,
+      "platform_default": true,
+      "version": 1,
```

Hey @kahowell - due to an issue with how the seeding gets run, the initial "version" needs to actually be set to `2`.
```suggestion
      "version": 2,
```